docs/extras/styling.md

```diff
@@ -120,7 +124,7 @@ Style your diagram elements ([ElkChildType][capellambse_context_diagrams.seriali
     diag = model.by_uuid("957c5799-1d4a-4ac0-b5de-33a65bf1519c").context_diagram
     diag.render_styles = dict(
         styling.BLUE_ACTOR_FNCS,
-        **{"junction": lambda _: {"fill": aird.RGB(220, 20, 60)}},
+        **{"junction": lambda _, __: {"fill": aird.RGB(220, 20, 60)}},
```

Comment:
The docs on this are a bit sparse:

- What arguments does this lambda get? Just giving them meaningful names in the docs (even though they're not used in this simple case) would already be enough for a start.
- What is it supposed to return? Which properties are valid, and who interprets them if it's not us?
- Are the lambda arguments expected to be extended again – should a lambda be prepared to receive additional args in the future?